tests/InputFiles/ScalarAdvection/Kuzmin2D.yaml

```diff
@@ -25,8 +17,8 @@ DomainCreator:
   Rectangle:
     LowerBound: [0.0, 0.0]
     UpperBound: [1.0, 1.0]
-    InitialRefinement: [3, 3]
-    InitialGridPoints: [3, 3]
+    InitialRefinement: [1, 1]
```

Comment:
Only commenting on this input file, but switching to subcell I'd expect you increase the number of points and decrease the lev. Eg lev 3->2, points 2->4. What's the reason for decreasing both? Is it that subcell is causing the tests to time out?

Reply:
At the first place (before adding Krivodonova and Kuzmin solution) it was lev 1 with points 2, I guess this is chosen to minimize the time taken for the execute test running. When I added `execute_check_output` test in #3405 for those two types of analytic solutions I needed to increase the resolution enough to ensure that the test is catching the real error norm (starts to decrease with increasing resolution), capturing the irregular shape of those two initial data. Since subcell executable currently cannot perform `execute_check_output` test, I rolled the resolutions back to the  minimal one.